src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -502,6 +505,9 @@ class SharedLinkSection extends React.Component<Props, State> {
         );
 
         if (!submitting) {
+            const getText = text => {
+                <span id={toggleDescriptionId}>{text}</span>;
+            };
```

Comment:
Just wondering, why are we wrapping the text with a span? I think according to Tooltip's implementation, the `text` prop should actually be a string

Reply:
nevermind, it seems to be currently set as type ReactNode, but it is used as a string here:
https://github.com/box/box-ui-elements/blob/master/src/components/tooltip/Tooltip.tsx#L254

I think we prefer to use strings though, so we don't have duplicate aria labels (aria-label and aria-desribedBy)

https://github.com/box/box-ui-elements/blob/master/src/components/tooltip/Tooltip.tsx#L289